libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -311,6 +311,13 @@ MQTTStatus_t MQTT_SerializeConnect( const MQTTConnectInfo_t * pConnectInfo,
 /**
  * @brief Get packet size and Remaining Length of an MQTT SUBSCRIBE packet.
  *
+ * This function must be called before #MQTT_SerializeSubscribe in order to
```

Comment:
in order to verify the size of --> in order to get the size of 

@p pPacketSize returned should be used to ensure that the size of the #MQTTFixedBuffer_t supplied to MQTT_SerializeSubscribe is at least this much.